tajo-core/src/main/java/org/apache/tajo/worker/TajoWorker.java

```diff
@@ -200,6 +200,7 @@ public void serviceInit(Configuration conf) throws Exception {
     historyReader = new HistoryReader(workerContext.getWorkerName(), this.systemConf);
 
     FunctionLoader.loadUserDefinedFunctions(systemConf);
+    HiveFunctionLoader.loadHiveUDFs(systemConf);
```

Comment:
It seems that TajoWorker also needs to check duplicated functions as TajoMaster. In addition, it will be better if the routine to load functions is consistent no matter where it is called from.